src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -134,6 +161,37 @@ describe('EntryFieldsComponent', () => {
     expect(toastrServiceStub.error).toHaveBeenCalled();
   });
 
+  it('displays error message when new hour entered is in the past of other entry', () => {
```

Comment:
I am not quite sure about what are you testing here. Correct me if I am wrong please:
```
Displays an error message when new entry has start_time before (start time/end time???) of active entry
```

Reply:
yes, it's the same, the name could be:
`Displays an error message when new entry has start_time before end time of last entry`